src/flick/like/tests.py

```diff
@@ -8,19 +8,17 @@
 from show.models import Show
 
 
-class ShowRatingsTests(TestCase):
```

Comment:
how come this shows as you deleting `test_show_rating.py`?

Reply:
dude idk